src/components/details/Cast.jsx

```
@@ -1,4 +1,4 @@
-/* eslint-disable react/prop-types */
+
 import React, { useRef } from 'react';
 
 export default function Cast({ details }) {
```

ESLint

[error] 'details' is missing in props validation

(react/prop-types)